Rakefile

```diff
@@ -2,25 +2,25 @@ require "bundler/gem_tasks"
 require "rspec/core/rake_task"
 
 namespace :spec do
-  task :db_drop do
+  task :db_drop => :environment do
```

Comment:
Is this right?  do you have an environment task?

Reply:
`Don't know how to build task 'environment' (see --tasks)`

Reply:
Seems like an overzealous rubocop "fix"